ground_type_protocol:
  right_lane: 0
  wrong_lane: 1
  partially_out: 2
  lost: 3

safety_status_protocol:
  fine: 1
  collision: -1